src/test/java/com/fryrank/controller/ReviewControllerTests.java

```diff
@@ -33,6 +37,9 @@ public class ReviewControllerTests {
     @InjectMocks
     ReviewController controller;
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
```

Comment:
You can remove this

Reply:
Done in [c52da6f](https://github.com/NickPriv/FryRankBackend/pull/62/commits/c52da6f116a93053a61ae696f15c2504a2a2a5c4).